runtime/packages/modules/timestamp/assembly/timestamp.ts

```diff
@@ -78,12 +74,11 @@ export class Timestamp{
      *  Gets the current time that was set. If this function is called prior 
      *  to setting the timestamp, it will return the timestamp of the previous block.
      */
-    static get(): u64 {
+    static get(): UInt64 {
         const now = Storage.get(Timestamp.SCALE_TIMESTAMP_NOW);
-        const res = now.isSome() ? (<ByteArray>now.unwrap()).values : [4, 0];
+        const res = now.isSome() ? (<ByteArray>now.unwrap()).values : [0, 0, 0, 0, 0, 0, 0, 0];
```

Comment:
Isn't this `UInt64(0)`? I think that it would be better to use it that way instead of hardcoding zeros. Maybe we can even add `UInt64.Zero` and `.One` to SCALE codec similarly to the `UInt128`